Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.yml

```diff
@@ -430,579 +430,579 @@ script:
+      description: Operator's mainframe login name.
       type: String
     - contextPath: TOPdesk.Operator.HasAttention
-      description: Operator's has attention
+      description: Operator's has attention.
       type: Boolean
     - contextPath: TOPdesk.Operator.Attention
-      description: Operator's attention
+      description: Operator's attention.
       type: Unknown
     - contextPath: TOPdesk.Operator.Comments
-      description: Operator's comments
+      description: Operator's comments.
       type: String
     - contextPath: TOPdesk.Operator.Installer
-      description: Operator's installer
+      description: Operator's installer.
       type: Boolean
     - contextPath: TOPdesk.Operator.FirstLineCallOperator
-      description: Operator's first line call operator
+      description: Operator's first line call operator.
       type: Boolean
     - contextPath: TOPdesk.Operator.SecondLineCallOperator
-      description: Operator's second line call operator
+      description: Operator's second line call operator.
       type: Boolean
     - contextPath: TOPdesk.Operator.ProblemManager
-      description: Operator's problem manager
+      description: Operator's problem manager.
       type: Boolean
     - contextPath: TOPdesk.Operator.ProblemOperator
-      description: Operator's problem operator
+      description: Operator's problem operator.
       type: Boolean
     - contextPath: TOPdesk.Operator.ChangeCoordinator
-      description: Operator's change coordinator
+      description: Operator's change coordinator.
       type: Boolean
     - contextPath: TOPdesk.Operator.ChangeActivitiesOperator
-      description: Operator's change activities operator
+      description: Operator's change activities operator.
       type: Boolean
     - contextPath: TOPdesk.Operator.RequestForChangeOperator
-      description: Operator's request for change operator
+      description: Operator's request for change operator.
       type: Boolean
     - contextPath: TOPdesk.Operator.ExtensiveChangeOperator
-      description: Operator's extensive change operator
+      description: Operator's extensive change operator.
       type: Boolean
     - contextPath: TOPdesk.Operator.SimpleChangeOperator
-      description: Operator's simple change operator
+      description: Operator's simple change operator.
       type: Boolean
     - contextPath: TOPdesk.Operator.ScenarioManager
-      description: Operator's scenario manager
+      description: Operator's scenario manager.
       type: Boolean
     - contextPath: TOPdesk.Operator.PlanningActivityManager
-      description: Operator's planning activity manager
+      description: Operator's planning activity manager.
       type: Boolean
     - contextPath: TOPdesk.Operator.ProjectCoordinator
-      description: Operator's project coordinator
+      description: Operator's project coordinator.
       type: Boolean
     - contextPath: TOPdesk.Operator.ProjectActiviesOperator
```

Comment:
@atossz Is it supposed to be `ProjectActiviesOperator` or `ProjectActivitiesOperator`? (Activies or Activi**ti**es?)

Reply:
Typo in the API of TOPdesk itself I think. The API documentation of TOPdesk also shows: projectActiviesOperator
without the extra T